src/components/buttons/Button.spec.jsx

```diff
@@ -54,6 +54,25 @@ test('icon', () => {
   expect(button.find('.sg-button__icon')).toHaveLength(1);
 });
 
+test('icon only', () => {
+  const icon = <span>:P</span>;
+  const button = shallow(
+    <Button icon={icon} iconOnly>
+      Some text
+    </Button>
+  );
+
+  expect(button.contains(icon)).toEqual(true);
+  expect(button.find('.sg-button--icon-only')).toHaveLength(1);
+  expect(button.find('.sg-button__icon')).toHaveLength(1);
+});
+
+test('icon only when no icon', () => {
+  const button = shallow(<Button iconOnly>Some text</Button>);
```

Comment:
is it possible to type it in a way that when you pass `iconOnly`, you must pass also `icon`?

Reply:
it's already typed but tests are flow-free

Reply:
yepp, but I think we don't need such test then? idk

Reply:
good point, removed